### PR TITLE
CFS Database Changes

### PR DESCRIPTION
**Pull request by gchauhan-aot**
# Description

Database changes for CFS file. Updating BE to add row to CFS table when payment method is one of the following: Ice pay, Cash and Cheque.
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] Tested manually

## Checklist

- [x] I have read the [CONTRIBUTING](CONTRIBUTING.md) doc
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have already been accepted and merged

## Further comments

---

Thanks for the PR!

Deployments, as required, will be available below:
- [Frontend](https://onroutebc-1393-frontend.apps.silver.devops.gov.bc.ca)
- [Vehicles](https://onroutebc-1393-vehicles.apps.silver.devops.gov.bc.ca/api)
- [Dops](https://onroutebc-1393-dops.apps.silver.devops.gov.bc.ca/api)
- [Scheduler](https://onroutebc-1393-scheduler.apps.silver.devops.gov.bc.ca/api)

Please create PRs in draft mode.  Mark as ready to enable:
- [Analysis Workflow](https://github.com/bcgov/onroutebc/actions/workflows/analysis.yml)

After merge, new images are promoted to:
- [Merge Workflow](https://github.com/bcgov/onroutebc/actions/workflows/merge.yml)

### CODE REVIEW

**Review by krishnan-aot — APPROVED** [review]

File: vehicles/src/common/enum/payment-method-type.enum.ts [hunk 1]
```diff
@@ -22,3 +22,20 @@ export type PaymentMethodTypeReport =
...
+  PaymentMethodType.ICEPAY | PaymentMethodType.CASH | PaymentMethodType.CHEQUE
+>;
+
+export const CfsPaymentMethodType = {
+  [PaymentMethodType.ICEPAY]: PaymentMethodType.ICEPAY,
+  [PaymentMethodType.CASH]: PaymentMethodType.CASH,
+  [PaymentMethodType.CHEQUE]: PaymentMethodType.CHEQUE,
+} as const;
+
+export const isCfsPaymentMethodType = (
```
Comment (praju-aot) [inline vehicles/src/common/enum/payment-method-type.enum.ts:37]: Please move the code to the payment helper/util or cfs helper/util

File: database/mssql/scripts/versions/v_27_ddl.sql [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+GO
+SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+GO
+BEGIN TRANSACTION
+GO
+
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+CREATE TABLE [permit].[ORBC_CFS_FILE_STATUS](
```
Comment (praju-aot) [inline database/mssql/scripts/versions/v_27_ddl.sql:18]: Rename to ORBC_CFS_FILE_STATUS_TYPE to confirm to MOTI DB

File: database/mssql/scripts/versions/v_27_ddl.sql [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+SET TRANSACTION ISOLATION LEVEL SERIALIZABLE
+GO
+BEGIN TRANSACTION
+GO
+
+IF @@ERROR <> 0 SET NOEXEC ON
+GO
+
+CREATE TABLE [permit].[ORBC_CFS_FILE_STATUS](	
+	[CFS_FILE_STATUS] [varchar] (10) NOT NULL,
```
Comment (praju-aot) [inline database/mssql/scripts/versions/v_27_ddl.sql:19]: Rename to CFS_FILE_STATUS_TYPE to confirm to Moti DB

File: vehicles/src/modules/permit-application-payment/payment/entities/cfs-transaction.entity.ts [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  @Column({
+    length: '30',
+    name: 'FILE_NAME',
+    nullable: true,
+  })
+  fileName: string;
+
+  @AutoMap()
+  @ApiProperty({
+    example: 'READY',
```
Comment (praju-aot) [inline vehicles/src/modules/permit-application-payment/payment/entities/cfs-transaction.entity.ts:42]: CfsFileStatus.READY

File: database/mssql/scripts/versions/revert/v_27_ddl_revert.sql [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+END TRY
+
+BEGIN CATCH
+  IF @@TRANCOUNT > 0 
+    ROLLBACK;
+  THROW
+END CATCH
+
+DECLARE @VersionDescription VARCHAR(255)
+SET @VersionDescription = 'Reverting initial creation of entities for Payment.'
```
Comment (praju-aot) [inline database/mssql/scripts/versions/revert/v_27_ddl_revert.sql:24]: Update the description to reflect CFS

File: database/mssql/test/versions/v_27_test.sh [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+parse_options "${USAGE}" ${@}
+
+# All database tests for database version 27 are run from this shell script.
+# TESTS_DIR variable set by the calling test-runner script.
+
+# Test 27.1 - verify that the transaction table exists
+TEST_27_RESULT=$(/opt/mssql-tools/bin/sqlcmd -U ${USER} -P "${PASS}" -S ${SERVER} -v DB_NAME=${DATABASE} -h -1 -i ${TESTS_DIR}/v_27_1_test.sql)
+
+if [[ $TEST_27_1_RESULT -eq 1 ]]; then
+    echo "Test 27.1 passed: All ORBC transaction tables exist"
```
Comment (praju-aot) [inline database/mssql/test/versions/v_27_test.sh:15]: Update the descriptions to reflect CFS

**Pull request merged**